modules/oe_theme_content_publication/oe_theme_content_publication.post_update.php

```diff
@@ -128,3 +128,23 @@ function oe_theme_content_publication_post_update_00007() {
   $entity = $entity_storage->createFromStorageRecord($config);
   $entity->save();
 }
+
+/**
+ * Update publication teaser view display.
+ */
+function oe_theme_content_publication_post_update_00008(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_publication') . '/config/post_updates/00008_update_teaser_view_display');
+  $display_values = $storage->read('core.entity_view_display.node.oe_publication.teaser');
+  $storage = \Drupal::entityTypeManager()->getStorage('entity_view_display');
+
+  // Take over teaser view display, regardless if it already exists or not.
+  $view_display = EntityViewDisplay::load($display_values['id']);
+  if ($view_display) {
+    $display = $storage->updateFromStorageRecord($view_display, $display_values);
+    $display->save();
+    return;
+  }
+
+  $display = $storage->createFromStorageRecord($display_values);
```

Comment:
Teaser view display is created in `oe_theme_content_publication_post_update_00004()`. If it was removed do we need to create it again? Looks like yes.

If yes, then `$config['_core']['default_config_hash'] = Crypt::hashBase64(serialize($config));` should be implemented prior to `$storage->createFromStorageRecord`. See oe_theme_content_news_post_update_00007().

Reply:
That's correct, @georgebaev you are working on it right?

Reply:
I added the config_hash related functionality.